### DISCUSSION TITLE
Request for Maintainer or Write Access on GitHub

### DISCUSSION THREAD

**Discussion by akhundubedullah** [Devasy23/NewsAI#45] [General]
I just reviewed the mentor list and found that I’ve been assigned to this project as a mentor. I’m excited to work with you! Could you please grant me maintainer or write access to the GitHub repository so I can get started?

Thank you!

**Comment by Devasy** [comment]
Hi @akhundubedullah , We are trying to get you started asap, but we'll have a meet first to give you understanding of the project and it's working, & discuss about it's projection we are keeping it towards

**Reply by akhundubedullah** [reply]
Ok got it!

**Reply by Devasy** [reply]
I've provided you triage role such that you can manage the repo PRS and issues.

We'll communicate further on LinkedIn or discord.
https://discord.gg/kV4ANf6x

**Reply by akhundubedullah** [reply]
Ok thanks
